build-support/wheel_build_x86_64/Dockerfile

```diff
@@ -8,3 +8,8 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
     ./aws/install && \
     rm -rf ./aws && \
     rm -f awscliv2.zip
+
```

Comment:
Even though we don't use this image afaict.

Reply:
Yeah, I created it for symmetry, and also to pre-install more useful stuff (e.g., rust), but could not get it to work in practice. Which is weird since its base image is the one we currently use. It's on my TODO list to figure out why.

Reply:
Maybe this PR will fix whatever that other issue was... :)